packages/esm-ward-app/src/home.comonent.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
This filename is spelled incorrectly.  Additionally, I'd rather not create components that have no basis in reality yet.  We have no design for a "Home" page in the ward app.  Get rid of this component